apps/orakul-starter/src/api/api.controller.ts

```diff
@@ -5,18 +5,18 @@ import { ScoutService } from '../scout/scout.service';
 
 @Controller('api')
 export class ApiController {
-  constructor(private scoutService: ScoutService) {}
+  constructor(private scoutService: ScoutService) { }
 
   @Get('/analysis')
   async fetchAnalysis() {
     try {
       const res = await this.scoutService.getAnalysis();
       return res;
     } catch (error) {
-      throw new BadRequestException(error.message, {
+      throw new BadRequestException(error.message, JSON.stringify({
```

Comment:
@MrSharpp , can you please explain why `JSON.stringify` is needed here?
BadRequestException constructor receives string and object.
https://docs.nestjs.com/exception-filters

Reply:
i can see new `BadRequestException(objectOrError?: any, description?: string): BadRequestException`
when going to its definition, I think the currently installed nest js doesn't have an overload for this method. and has 
only string as its 2nd argument.